curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a24015fe4fbd02a16cbbe7.md

```diff
@@ -0,0 +1,346 @@
+---
+id: 64a24015fe4fbd02a16cbbe7
+title: Step 36
+challengeType: 0
+dashedName: step-36
+---
+
+# --description--
+
+NEED HELP ! The test is not passing, any ideas? 
```

Comment:
The regex is fine, I tested that. There seems to be some issue with the order of execution. I have also modified the regex and tested it, but it still doesn't pass.